packages/voting/graphql/schema-types.js

```diff
@@ -57,18 +60,30 @@ type VotingStatsCount {
   options: [VotingOptionResult!]!
 }
 
+type VotingMembershipRequirement {
+  membershipTypeId: ID!
+  createdBefore: DateTime!
+}
+
 input VotingInput {
   name: String!
   description: String!
   slug: String!
   beginDate: DateTime!
   endDate: DateTime!
   options: [VotingOptionInput!]!
+  allowedMemberships: [VotingMembershipRequirementInput!]
+  allowedRoles: [String!]
 }
 
 input VotingOptionInput {
   name: String!
   label: String
   description: String
 }
+
+input VotingMembershipRequirementInput {
```

Comment:
100% equal to `VotingMembershipRequirement`. Maybe just use that directly?

Reply:
one is a `type` the other is an `input`